homeassistant/components/livisi/const.py

```diff
@@ -2,11 +2,18 @@
 import logging
 from typing import Final
 
+import voluptuous as vol
+
 LOGGER = logging.getLogger(__package__)
 DOMAIN = "livisi"
-
 CONF_HOST = "host"
 CONF_PASSWORD: Final = "password"
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_PASSWORD): str,
+    }
+)
```

Comment:
Even though it is a constant, it is specific to config-flow and should be kept in the config-flow module.

Reply:
I refactored this, thank you.